partcad/src/partcad/wrappers/wrapper_obj.py

```
@@ -0,0 +1,66 @@
+
+import os
+import sys
+
+from OCP.TopoDS import TopoDS_Compound, TopoDS_Builder
+from OCP.BRepBuilderAPI import BRepBuilderAPI_MakeFace, BRepBuilderAPI_MakePolygon
+from OCP.gp import gp_Pnt
+
+sys.path.append(os.path.dirname(__file__))
+from . import wrapper_common
+
+def process(path, request):
```

Ruff (0.8.2)

Missing return type annotation for public function `process`

(ANN201)